src/pages/services.astro

```diff
@@ -0,0 +1,109 @@
+---
+import {useLocalePages, useServiceTranslations, useTranslations} from '../i18n/utils';
+import Layout from '../layouts/Layout.astro';
+import Section from '../components/Section.astro';
+import {Icon} from 'astro-icon/components';
+import {getEntry} from 'astro:content';
+import {Image} from 'astro:assets';
+import type {DataEntryMap} from 'astro:content';
+
+const p = useLocalePages(Astro.currentLocale);
+
+const st = useServiceTranslations(Astro.currentLocale);
+const t = useTranslations(Astro.currentLocale);
+
+const servicePrefixes = ['esg', 'portals', 'processing', 'integration'] as const;
```

Comment:
why did you create markup files with contents for the references but prefixed translations for the services? why not make it uniformly?

Reply:
I'm not sure how we can make the more complex styling, then.